packages/application-shell/src/components/quick-access/create-commands.js

```diff
@@ -18,7 +18,7 @@ export default function createCommands({
       applicationContext.permissions &&
       featureToggles.canViewDashboard &&
       hasSomePermissions(
```

Comment:
hasSomePermissions calls hasPermission which determines itself if the passed permissions (e.g. a view) is implied by a manage.